bindings/rust/evmc-declare/src/lib.rs

```diff
@@ -268,8 +267,7 @@ fn build_static_data(names: &VMNameSet, metadata: &VMMetaData) -> proc_macro2::T
 
 /// Takes a capabilities flag and builds the evmc_get_capabilities callback.
 fn build_capabilities_fn(capabilities: u32) -> proc_macro2::TokenStream {
-    let capabilities_literal =
-        LitInt::new(capabilities as u64, IntSuffix::U32, capabilities.span());
+    let capabilities_literal = LitInt::new(&capabilities.to_string(), capabilities.span());
```

Comment:
I wonder if `LitInt` copies the string or at least `quote` will just print it below and we don't need to take care about lifetime.

Reply:
Will at least make the string its own variable to ensure its lifetime.

Reply:
I am unsure because it takes a reference to an owned type but it also wouldn't make sense in this crate to expect the user to keep the members in scope.